keyboards/kbdfans/kbd67/mkiirgb/keymaps/codecoffeecode/keymap.c

```diff
@@ -0,0 +1,56 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  /* Base layer
+   * ,--------------------------------------------------------------------------------------------------.
+   * |Esc ~|  1  |  2  |  3  |  4  |  5  |  6  |  7  |  8  |  9  |  0  |  -  |  =  |  Backspace  | Home |
+   * |-------------------------------------------------------------------------------------------+------+
+   * | Tab    |  Q  |  W  |  E  |  R  |  T  |  Y  |  U  |  I  |  O  |  P  |  [  |  ]  |    \     | End  |
+   * |-------------------------------------------------------------------------------------------+------+
+   * | CtrlCaps |  A  |  S  |  D  |  F  |  G  |  H  |  J  |  K  |  L  |  ;  |  '  |     Enter    | PgUp |
+   * |-------------------------------------------------------------------------------------------+------+
+   * | Shift      |  Z  |  X  |  C  |  V  |  B  |  N  |  M  |  ,  |  .  |  /  |   MO(2)    | Up  | PgDn |
+   * +-------------------------------------------------------------------------┬---┬-------------+------+
+   * | LCtrl | LGUI | LAlt |               Space                 | Btn2 | MO(1)|   | Left  |Down | Right|
+   * `-------------------------------------------------------------------------┘   └-------------+------´
+   */
+    [0] = LAYOUT_65_ansi_blocker( /* Base */
+        KC_GESC,       KC_1,     KC_2,     KC_3,  KC_4,  KC_5,  KC_6,    KC_7,  KC_8,    KC_9,     KC_0,     KC_MINS,  KC_EQL,  KC_BSPC,   KC_HOME,\
+        KC_TAB,        KC_Q,     KC_W,     KC_E,  KC_R,  KC_T,  KC_Y,    KC_U,  KC_I,    KC_O,     KC_P,     KC_LBRC,  KC_RBRC, KC_BSLASH, KC_END, \
+        CTL_T(KC_CAPS),KC_A,     KC_S,     KC_D,  KC_F,  KC_G,  KC_H,    KC_J,  KC_K,    KC_L,     KC_SCLN,  KC_QUOT,           KC_ENT,    KC_PGUP,\
+        KC_LSFT,       KC_Z,     KC_X,     KC_C,  KC_V,  KC_B,  KC_N,    KC_M,  KC_COMM, KC_DOT,   KC_SLSH,  MO(2),             KC_UP,     KC_PGDN,\
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.